src/DocumentFormat.OpenXml/GeneratedCode/schemas_microsoft_com_office_drawing_2016_11_diagram.g.cs

```diff
@@ -0,0 +1,376 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Drawing;
+using DocumentFormat.OpenXml.Drawing.Diagrams;
+using DocumentFormat.OpenXml.Framework;
+using DocumentFormat.OpenXml.Framework.Metadata;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Validation.Schema;
+using System;
+using System.Collections.Generic;
+using System.IO.Packaging;
+
+namespace DocumentFormat.OpenXml.Office2019.Drawing.Diagram11
+{
+    /// <summary>
+    /// <para>Defines the NumberDiagramInfoList Class.</para>
+    /// <para>This class is available in Office 2007 or above.</para>
```

Comment:
if this is a new class, how can it be available in Office 2007 or above?

Reply:
I understand what happened. I will have to rework some code to get this fixed. Thanks for finding it.